### PR TITLE
added excelserviceprovider

### PR DESCRIPTION
**Pull request by serdarekremcakir**

### CODE REVIEW

File: src/Http/Controller/Admin/CategoryController.php [hunk 1]
```diff
@@ -7,10 +7,10 @@
 use Anomaly\Streams\Platform\Model\Cats\CatsCategoryEntryTranslationsModel;
 use Anomaly\Streams\Platform\Ui\Form\FormBuilder;
 use Illuminate\Http\Request;
-use Illuminate\Http\UploadedFile;
 use Illuminate\Support\Facades\DB;
 use Illuminate\Support\Str;
 use League\Flysystem\MountManager;
+use Maatwebsite\Excel\ExcelServiceProvider;
```
Comment (fatihalp) [inline src/Http/Controller/Admin/CategoryController.php:13]: We can make it dynamic

File: src/Http/Controller/Admin/CategoryController.php [hunk 2]
```diff
@@ -274,6 +274,7 @@ public function import(FormBuilder $builder, FileRepositoryInterface $fileReposi
         if (request()->action == "save" and $file = $fileRepository->find(request()->file)) {
             if ($file->extension === 'xls' || $file->extension === 'xlsx') {
                 $pathToFolder = "/storage/streams/" . app(Application::class)->getReference() . "/files-module/local/ads_excel/";
+                app()->register(ExcelServiceProvider::class);
```
Comment (fatihalp) [inline src/Http/Controller/Admin/CategoryController.php:277]: we can load here @serdarekremcakir

File: src/Http/Controller/Admin/CategoryController.php (hunk 1 above)
Comment (fatihalp) [reply src/Http/Controller/Admin/CategoryController.php:13]: it's not provider then it accaptable

**Review by fatihalp — APPROVED** [review]

File: src/Http/Controller/Admin/CategoryController.php (hunk 2 above)
Comment (spektra2147) [reply src/Http/Controller/Admin/CategoryController.php:277]: please add CatsServiceProvider.php. You can use `protected $provider  = []`;

**Pull request closed without merging**